src/components/tables/RowActions/Delete/Confirmation.tsx

```diff
@@ -0,0 +1,26 @@
+import { Alert, AlertTitle, Box, Stack } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+interface DeleteConfirmationprops {
```

Comment:
The leading _p_ in _props_ is should be capitalized: `DeleteConfirmationProps`. That said, why not opt for the generic `Props` here?